docs/README.md

```
@@ -0,0 +1,14 @@
+# Winterfell
+
+I shall wear no crowns and win no glory.
+
+TODO: Write intro
+
+
+## Usage
+TODO: Write usage instructions
+
+
+<br/><br/>
```

Markdownlint (0.37.0)

Element: br
Inline HTML

(MD033, no-inline-html)